elements/drawtools/src/style.eox.js

```diff
@@ -15,3 +15,80 @@ button.polygon:before {
   content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='%23004170' viewBox='0 0 24 24'%3E%3Ctitle%3Eshape-polygon-plus%3C/title%3E%3Cpath d='M17,15.7V13H19V17L10,21L3,14L7,5H11V7H8.3L5.4,13.6L10.4,18.6L17,15.7M22,5V7H19V10H17V7H14V5H17V2H19V5H22Z' /%3E%3C/svg%3E");
 }
 `;
+
+export const styleListEOX = `
+  * {
+    font-family: Roboto, sans-serif;
+  }
+  ul {
+    padding: 0;
+  }
+  ul ul {
+    padding-left: 48px;
+  }
+  li:hover {
+    background: #f0f5f9;
+  }
+  li.selected {
+    background: #f0f5f9;
+  }
+  li {
+    list-style: none;
+    padding: 4px;
+  }
+  li {
+    border-bottom: 1px solid #0041703a;
+  }
+  li:first-child {
+    border-top: 1px solid #0041703a;
+  }
+  li.sortable-chosen {
+    background: #eeea;
+  }
+  li.sortable-drag {
+    opacity: 0;
+  }
+  li.sortable-ghost {
+  }
+  eox-drawtools-list {
+    width: 100%;
+  }
+  .list {
+    width: 100%;
+    align-items: center;
+    justify-content: space-between;
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    font-size: small;
+    gap: 10px;
+
+  }
+  label {
+    gap: 10px;
+  }
+  label, span {
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+    font-size: small;
+    flex-grow: 1;
+  }
+  .list .remove-icon::before {
+    content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='%23ff0000' viewBox='0 0 24 24'%3E%3Ctitle%3Edelete-outline%3C/title%3E%3Cpath d='M6,19A2,2 0 0,0 8,21H16A2,2 0 0,0 18,19V7H6V19M8,9H16V19H8V9M15.5,4L14.5,3H9.5L8.5,4H5V6H19V4H15.5Z' /%3E%3C/svg%3E");
```

Comment:
Here it would probably best to harmonize things and use the same icon as the `discard` button (see L11).

Reply:
Done ✅